test/models/patient_test.rb

```diff
@@ -152,6 +152,18 @@ class PatientTest < ActiveSupport::TestCase
       assert_equal summary[:pledged].count, 1
       assert_equal summary[:sent].count, 1
     end
+
+    it "should support a different start of week" do
+      # if the week did NOT start on thursday then patients would not be counted
```

Comment:
I'm not sure I quite follow what this test is proving -- I think that this is creating a fund_pledged_at on a Saturday 

I would suggest having the config_value be `Saturday` and pledging to @patient_2 on Jan 19, 1973 (so we can prove that Saturday -> Friday gets picked up). A good practice I learned from @lomky is that when you're doing dates or times with edges, do one on the included side and one on the filtered side, and I thnk that's an appropriate rule of thumb here.

I would also suggest this comment which I think is a hair clearer (and which means whoever touches this next won't have to google what day Jan 20 was):

```suggestion
      # For a start day of Saturday, Saturday should be counted; the Friday before should not
      # Jan 19, 1973 is a Friday; Jan 20 is a Saturday
```

Reply:
Addressed.